src/js/claims-status/components/appeals-v2/Timeline.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import moment from 'moment';
+
+import { getEventContent } from '../../utils/appeals-v2-helpers';
+import CurrentStatus from './CurrentStatus';
+
+/**
+ * Timeline is in charge of the past events and current status.
+ */
+class Timeline extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expanded: false
+    };
+  }
+
+  getPastEvents = () => {
+    return this.props.events.map((event, index) => {
+      const { title, description, liClass } = getEventContent(event);
+      return (
+        <li key={index} role="presentation" className={`process-step ${liClass}`}>
+          <h3>{title || 'Title here'}</h3>
+          <span className="appeal-event-date">on {moment(event.date, 'YYYY-MM-DD').format('MMMM d, YYYY')}</span>
+          <p>{description}</p>
+        </li>
+      );
+    });
+  }
+
+  toggleExpanded = () => {
+    this.setState({ expanded: !this.state.expanded });
+  }
+
+  render() {
+    const eventList = this.state.expanded ? this.getPastEvents() : [];
+    const dateRange = this.props.events[0] ? `${moment(this.props.events[0].date, 'YYYY-MM-DD').format('MMMM d, YYYY')} - ${moment(this.props.events[this.props.events.length - 1].date, 'YYYY-MM-DD').format('MMMM d, YYYY')}` : '';
```

Comment:
Since we use this format throughout in at least 3 places, do you think it might be worth it to just extract this formatting into its own little function? E.g., 
`format = (rawDate) => {
    // do some formatting with moment...
    return formattedDate
}`

Might make this easier to read. 🤷‍♂️

Reply:
Yeah, I thought about it. I'll go ahead and make that happen. 👍